website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,176 @@
+With no constraints defined in provisioners and none requested from pods being deployed, Karpenter on AWS is free to choose from the entire universe of features available from EC2.
+Nodes can be created using any instance type and run in any zones.
+On-demand or spot instances can be used.
+
+Most of the same constraints a Karpenter cluster operator can set up on a provisioner can be used by an application developer in a podspec to request particular features from a node.
+Refer to the description of Karpenter constraints in the Application Developer section below for details.
+
+#### Scheduling
+
+To avoid conflicts, Karpenter only schedules pods that the Kubernetes scheduler has marked unschedulable.
+Karpenter optimistically creates the Node object and binds the pod.
+This stateless approach helps to avoid race conditions and improves performance.
+If something is wrong with the launched node, Kubernetes will automatically migrate the pods to a new node.
+
+Once Karpenter brings up a node, that node is available for the Kubernetes scheduler to schedule pods on it as well.
+This is useful if there is additional room in the node due to imperfect packing shape or because workloads finish over time.
+
+#### Cloud provider
+Karpenter makes requests to provision new nodes to the associated cloud provider.
+The first supported cloud provider is AWS, although Karpenter is designed to eventually work with other cloud providers as well.
+Separating Kubernetes and AWS-specific settings allows Karpenter a clean path to integrating with other cloud providers.
+
+While using Kubernetes well-known labels, the provisioner can set some values that are specific to the cloud provider.
+So, for example, to include a certain instance type, you could use the Kubernetes label `node.kubernetes.io/instance-type`, but set its value to an AWS instance type (such as `m5.large` or `m5.2xlarge`).
+
+Choosing spot instances is one feature that is currently specific to AWS, but one that could be generalized to other providers.
+Furhter, cloud providers may choose to implement support for additional vendor specific labels like `node.k8s.aws/capacity-type`.
+See [AWS labels](/docs/cloud-providers/aws/) for details.
+
+#### Kubernetes cluster autoscaler
+Like Karpenter, [Kubernetes Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) is
+designed to add nodes when requests come in to run pods that cannot be met by current capacity.
+Cluster autoscaler is part of the Kubenetes project, with implementations by most major Kubernetes cloud providers.
+By taking a fresh look at provisioning, Karpenter offers the following improvements:
+
+* **Designed to handle the full flexibility of the cloud**:
+Karpenter has the ability to efficiently address the full range of instance types available through AWS.
+Cluster autoscaler was not originally built with the flexibility to handle hundreds of instance types, zones, and purchase options.
+
```

Comment:
```suggestion

**Group-less node provisioning**:
Karpenter manages each instance directly, without use of additional orchestration mechanisms like node groups. This enables it to retry in milliseconds instead of minutes when capacity is unavailable. It also allows Karpenter to leverage diverse instance types, availability zones, and purchase options without creation of hundreds of node groups.

```